### ISSUE TITLE
Component reuse for webapp

### ISSUE THREAD

**Issue by arben777** [ndugdale/osrs-web-greeter-theme#6]
Hey Ndugdale I would really like to get in contact. Can you please contact me via twitter @aXXXXX(omitted) or at email aXXXXX(omitted)@me.com and let me know here when you do. I will of course delete this issue. I just have a question about this repo and would really appreciate your swift help. 

Thank you

**Comment by arben777** [comment]
Understood!

So from my understanding this code allows for the use of the OSRS login page to be used for a compute's login or for a computer's application login, and its use of react /web app components is for design and styling.

I am wanting to have this login page be used as a 'Preloader' / entrance page for a webapp and was having some trouble sorting out how to strip away the OS related code to achieve really just a front end webpage of this OSRS login. For full transparency I was trying to authentication / extra functionality like the showing or error, and demo ended pages so that this would just serve as almost an 'Enter' button for website. It seemed like an easy task but has been giving me some trouble. 

Thoughts on how much of a shift in this code it really is to achieve this? Also my main webpage is in NextJS using base JS not JSX (which wasn't hard to change) but there are unexpected react errors and such coming up. 

Obviously I expect no full or in depth answer for how to do such a conversion from this project you have graciously shared to the entrance style 'login' page I mention, but would love any insight or thoughts on how I may be progress or achieve this. 

TLDR: How strip this to just web app page? and in nextjs possible? no actually functionality other than 'enter' button for a site. 

<3
Thanks, I appreciate your time

**Comment by arben777** [comment]
Amazing. This was truly helpful. I did take the approach of copying over components to my NextJS project to avoid unnecessary and unique build scripts. 

Understood and very helpful point about the setting of the dm value "triggering" the full login functionality where without it the codes / else statements stand to be the web demo version. 

Again I appreciate the depth and the time you took in these replies as well as with this repo in general.

Cheers! 🥂
